.github/workflows/pages.yaml

```diff
@@ -11,7 +11,11 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v4
       - name: run for all SDKs
-        run: cd test-harness && go run ./cmd/web5-test-harness many sdks/* && mv _site ../_site
+        run: |
+          git clone https://github.com/TBD54566975/web5-js.git
```

Comment:
nit: let's favor ssh checkout over https

Reply:
this results in a failure ([ci output](https://github.com/TBD54566975/sdk-development/actions/runs/7022580060/job/19107178448?pr=68)):

<img width="511" alt="image" src="https://github.com/TBD54566975/sdk-development/assets/139806439/a37bc73e-3030-40cc-856f-e514ffcf43c3">